tensorrt_llm/_torch/models/modeling_utils.py

```diff
@@ -665,6 +724,10 @@ def filter_weights(prefix, weights: Dict):
     for name, module in tqdm(list(model.named_modules()),
                              desc="Loading weights"):
         if len(module._parameters) > 0:
+            # skip load weights if module is in skip_modules
+            if any(skip_module in name for skip_module in skip_modules):
+                continue
+
```

Comment:
@yuxianq Can you confirm if this is correct?

Reply:
This is needed because in one-engine approach, draft model is appended at the end of the target model as "additional layers". Therefore, we must skip the weight loading for the draft model when we load the weights for the target model.

Reply:
Please add some comments in the code. It's very hard to understand the intention of this code otherwise.